crates/lang/codegen/src/generator/events.rs

```diff
@@ -89,7 +89,7 @@ impl<'a> Events<'a> {
             proc_macro2::Ident::new("__ink_EventBase", Span::call_site());
         quote! {
             #[allow(non_camel_case_types)]
-            #[derive(::scale::Encode, ::scale::Decode)]
+            #[derive(::scale::Encode, ::scale::Decode, Debug)]
```

Comment:
As a general tip, inside of macros we aim to use full paths in case the code where the macro is be called has defined their own `Debug` type/trait (in which case this wouldn't work as expected) 

```suggestion
            #[derive(::scale::Encode, ::scale::Decode, ::core::fmt::Debug)]
```

Reply:
Thanks for the tip